examples/bigcommerce/pages/[[...path]].tsx

```diff
@@ -19,7 +19,7 @@ export async function getStaticProps(
   if (!('props' in makeswiftResult)) return makeswiftResult
 
   const products = await getProducts()
-  const product = await getProduct()
```

Comment:
We know that this will be default product so we might as well be explicit about it